cmd/deploy/deploy.go

```diff
@@ -505,25 +507,20 @@ func onlyDeployEndpointsFromServicesToDeploy(endpoints model.EndpointSpec, servi
 	}
 }
 
-func onlyDeployVolumesFromServicesToDeploy(stack *model.Stack, servicesToDeploy map[string]bool) {
-
-	volumesToDeploy := map[string]bool{}
+func getVolumesToDeployFromServicesToDeploy(stack *model.Stack, servicesToDeploy map[string]bool) []string {
```

Comment:
Can we get them on the stack package? I think that if we get here it can cause some problems when running `okteto stack deploy`. That way we don't need to create a new option